pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <groupId>com.github.cafdataprocessing</groupId>
     <artifactId>worker-document-aggregator</artifactId>
-    <version>6.1.0-SNAPSHOT</version>
+    <version>7.0.0-SNAPSHOT</version>
```

Comment:
Version number changes won't be required when we undo making this a breaking change.